mod_cluster/extension/src/main/resources/schema/jboss-as-mod-cluster_4_0.xsd

```diff
@@ -369,6 +377,13 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="module" type="xs:string">
+                    <xs:annotation>
+                        <xs:documentation>
+                            Module name to load the load metric class from.
```

Comment:
"Module name from which to load the load metric class."

Reply:
Fixed.